### ISSUE TITLE
Detailed annotations are hidden by default when printing

### ISSUE THREAD

**Issue by 6a6d74** [w3c/csvw#380]
csv2json and csv2rdf specs now have some java script magic in place to hide the detailed annotations for the examples. This is great. _But_ when you print these specs, those details remain hidden (and obviously can't be expanded in print form!).

It would be great to be able to set the default behaviour for print media to "expanded" whilst the default for the browser remains "hidden".

**Comment by iherman** [comment]
Just to "report" (maybe somebody has a good idea): I tried to use the `windows.matchMedia` call in the initialization part, as proposed by @gkellogg. The way I did it was something like

```
match = window.matchMedia("screen");
if( match.matches ) { ...do all the init stuff }
else { ... make the buttons invisible }
```

Which works for the screen as before. However, when (on a Mac) saving the file into PDF, it ignores this insofar as it behaves as if the media was a screen. This may be a Mac specificity, but it beats the purpose...

I will also ask around the W3C staff, maybe somebody has a good idea. And still looking...
